src/Storage/Storage.Management/help/Restore-AzDataLakeGen2DeletedItem.md

```diff
@@ -0,0 +1,233 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Storage.dll-Help.xml
+Module Name: Az.Storage
+online version: https://docs.microsoft.com/powershell/module/az.storage/restore-azdatalakegen2deleteditem
+schema: 2.0.0
+---
+
+# Restore-AzDataLakeGen2DeletedItem
+
+## SYNOPSIS
+Restores a deleted file or directory in a filesystem.
+
+## SYNTAX
+
+### ReceiveManual (Default)
+```
+Restore-AzDataLakeGen2DeletedItem [-FileSystem] <String> [-Path] <String> [-DeletionId] <String> [-AsJob]
+ [-Context <IStorageContext>] [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm]
+ [<CommonParameters>]
+```
+
+### ItemPipeline
+```
+Restore-AzDataLakeGen2DeletedItem -InputObject <AzureDataLakeGen2DeletedItem> [-AsJob]
+ [-Context <IStorageContext>] [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm]
+ [<CommonParameters>]
+```
+
+## DESCRIPTION
+The **Restore-AzDataLakeGen2DeletedItem** cmdlet restores a deleted file or directory in a filesystem in an Azure storage account.
+This cmdlet only works if Hierarchical Namespace is enabled for the Storage account. This kind of account can be created by run "New-AzStorageAccount" cmdlet with "-EnableHierarchicalNamespace $true".
+
+## EXAMPLES
+
+### Example 1: List all deleted files or directories from a Filesystem, and restore them by pipeline
+```
+PS C:\> $items = Get-AzDataLakeGen2DeletedItem -FileSystem "filesystem1" 
+
+PS C:\> $items
+
+   FileSystem Name: filesystem1
+
+Path                 DeletionId           DeletedOn            RemainingRetentionDays
+----                 ----------           ---------            ----------------------
+dir0/dir1/file1      132658816156507617   2021-05-19 07:06:55Z 3                     
+dir0/dir2            132658834541610122   2021-05-19 07:37:34Z 3                    
+dir0/dir2/file3      132658834534174806   2021-05-19 07:37:33Z 3   
+
+PS C:\> $items | Restore-AzDataLakeGen2DeletedItem 
+
+   FileSystem Name: filesystem1
+
+Path                 IsDirectory  Length          LastModified         Permissions  Owner                Group               
+----                 -----------  ------          ------------         -----------  -----                -----               
+dir0/dir1/file1      False        1024            2021-05-19 07:06:39Z rw-r-----    $superuser           $superuser          
+dir0/dir2            True                         2021-05-19 07:06:37Z rwxr-x---    $superuser           $superuser          
+dir0/dir2/file3      False        1024            2021-05-19 07:06:42Z rw-r-----    $superuser           $superuser
+```
```

Comment:
This example has multiple output, might need add 
`<!-- Skip: Output cannot be splitted from code -->`

Reply:
Done